core/runtime/src/main/java/io/quarkus/runtime/configuration/ApplicationPropertiesConfigSource.java

```diff
@@ -54,19 +56,25 @@
 
     public static final class InJar extends ApplicationPropertiesConfigSource {
         public InJar() {
-            super(openStream(), 250);
+            super(openStream(null), 260);
```

Comment:
Shouldn't we keep the same ordinal?

Reply:
I'm going to follow up your sugestions. 

I just keet the same pattern to add +10 for each one. But you're right. I would be better to not change the ordinal and only add +1.